ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java

```diff
@@ -370,19 +375,16 @@ private String createZoneSQL() {
         DEFAULT_COLUMNAR_PROFILE_NAME :
         secondaryStorageProfile;
 
-    String storageProfiles = useColumnar ?
-        String.join(",", storageProfile, secondaryStorageProfile) :
-        storageProfile;
-
-    String paramStorageProfiles = String.format("STORAGE_PROFILES='%s'", storageProfiles);
+    String paramStorageProfile = String.format("STORAGE PROFILE '%s'", storageProfile);
+    String paramSecondaryStorageProfile = String.format("SECONDARY STORAGE PROFILE '%s'", secondaryStorageProfile);
     String paramReplicas = replicas.isEmpty() ? "" : "replicas=" + replicas;
     String paramPartitions = partitions.isEmpty() ? "" : "partitions=" + partitions;
-    String params = Stream.of(paramStorageProfiles, paramReplicas, paramPartitions)
+
+    String params = Stream.of(paramStorageProfile, paramSecondaryStorageProfile, paramReplicas, paramPartitions)
```

Comment:
We should not create a zone with `SECONDARY STORAGE PROFILE` if `useColumnar` is `false.

Reply:
My bad, zone creation reverted